girder/lib/mongodb_proxy.py

```diff
@@ -0,0 +1,145 @@
+"""
```

Comment:
What do you think about changing the "lib" dir to "ext" (or "external" or "tpl") instead? Seeing "lib" in that location might make people assume it's girder library code in there.